scipy/linalg/flapack_pos_def_tri.pyf.src

```diff
@@ -83,3 +83,65 @@ subroutine <prefix2c>pttrs(lower, n, nrhs, d, e, b, ldb, info)
+    ! provided.
+    callstatement (*f2py_func)(fact, &n, &nrhs, d, e, df, ef, b, &ldb, x, &ldx, &rcond, ferr, berr, work, &info)
+    callprotoargument char*, int*, int*, <ctype2>*, <ctype2>*, <ctype2>*, <ctype2>*, <ctype2>*, int*, <ctype2>*, int*,  <ctype2>*, <ctype2>*, <ctype2>*, <ctype2>*, int*
+
+    character optional, intent(in) :: fact = 'N'
+    integer intent(hide), depend(d) :: n = len(d)
+    integer intent(hide), depend(b) :: nrhs = shape(b, 1)
+    <ftype2> intent(in), dimension(n) :: d
+    <ftype2> intent(in), depend(n), dimension(max(0, n-1)) :: e
+    <ftype2> optional, intent(in,out), depend(n), dimension(n) :: df
+    <ftype2> optional, intent(in,out), depend(n), dimension(max(0, n-1)) :: ef
+    <ftype2> intent(in), dimension(ldb, nrhs) :: b
+    integer intent(hide), depend(b) :: ldb = max(1, shape(b, 0))
+    <ftype2> intent(out), dimension(ldx, nrhs) :: x
+    integer intent(hide), depend(n) :: ldx = n
+    <ftype2> intent(out) :: rcond
+    <ftype2> intent(out), depend(nrhs), dimension(nrhs) :: ferr
+    <ftype2> intent(out), depend(nrhs), dimension(nrhs) :: berr
+    <ftype2> intent(hide,cache), depend(n), dimension(2*n) :: work
+    integer intent(out) :: info
+
+end subroutine <prefix2>ptsvx
+
+
+subroutine <prefix2c>ptsvx(fact, n, nrhs, d, e, df, ef, b, ldb, x, ldx, rcond, ferr, berr, work, rwork, info)
+    ! DPTSVX uses the factorization A = L*D*L**T to compute the solution
+    ! to a real system of linear equations A*X = B, where A is an N-by-N
+    ! symmetric positive definite tridiagonal matrix and X and B are
+    !N-by-NRHS matrices.
+    !
+    ! Error bounds on the solution and a condition estimate are also
+    ! provided.
+    callstatement (*f2py_func)(fact, &n, &nrhs, d, e, df, ef, b, &ldb, x, &ldx, &rcond, ferr, berr, work, rwork, &info)
+    callprotoargument char*, int*, int*, <ctype2>*, <ctype2c>*, <ctype2>*, <ctype2c>*, <ctype2c>*, int*, <ctype2c>*, int*,  <ctype2>*, <ctype2>*, <ctype2>*, <ctype2c>*, <ctype2>*, int*
+
+    character optional, intent(in) :: fact = 'N'
+    integer intent(hide), depend(d) :: n = len(d)
+    integer intent(hide), depend(b) :: nrhs = shape(b, 1)
+    <ftype2> intent(in), dimension(n) :: d
+    <ftype2c> intent(in), depend(n), dimension(max(0, n-1)) :: e
+    <ftype2> optional, intent(in,out), depend(n), dimension(n) :: df
+    <ftype2c> optional, intent(in,out), depend(n), dimension(max(0, n-1)) :: ef
+    <ftype2c> intent(in), dimension(ldb, nrhs) :: b
+    integer intent(hide), depend(b) :: ldb = max(1, shape(b, 0))
+    <ftype2c> intent(out), dimension(ldx, nrhs) :: x
+    integer intent(hide), depend(n) :: ldx = n
+    <ftype2> intent(out) :: rcond
+    <ftype2> intent(out), depend(nrhs), dimension(nrhs) :: ferr
+    <ftype2> intent(out), depend(nrhs), dimension(nrhs) :: berr
+    <ftype2c> intent(hide,cache), depend(n), dimension(2*n) :: work
+    <ftype2> intent(hide,cache), depend(n), dimension(2*n) :: rwork
```

Comment:
These should be of dimension `n` not `2n`. It's different from the real types where you have one array of dimension(2*n).
```suggestion
    <ftype2c> intent(hide,cache), depend(n), dimension(n) :: work
    <ftype2> intent(hide,cache), depend(n), dimension(n) :: rwork
```